go.mod

```diff
@@ -3,13 +3,16 @@ module github.com/cheqd/cheqd-node
 go 1.17
 
 require (
+	filippo.io/edwards25519 v1.0.0-beta.2
```

Comment:
It's needed just for checking ED25519 key, just for confirming, that publickey represents correct point.

Reply:
So we had two options here:
* Try signing and catch panic;
* Use `filippo.io/edwards25519` <- we choose this option.

Do you think there is a better way to do it?

Reply:
One more thing: if there is a bug in the library (in which I doubt because it's an extended copy of the internal one) and an invalid key passes static validation, signatures will be checked using standard `ed25519` library and the request will fail.